### ISSUE TITLE
Am I able to rebind keybinds?

### ISSUE THREAD

**Issue by Whiskah** [insthync/shooter-io-playfab-client-demo#4]
Hello, I see there is an input manager script in the package which seems to take the inputs from Unitys Input Manager, does this mean I am unable to get/set and rebind the controls that are used in this without having to write my own custom input manager?

**Comment by insthync** [comment]
No, it has no key binding setting yet, but I will add it, follow the changes here https://github.com/insthync/unity-camera-and-input

Now it has `Input Setting Manager` component which will be used to store default key binding setting, so you can set key-binding by the component instead by Unity's Input manager (from project settings).

**Comment by Whiskah** [comment]
Awesome thankyou, looking at the changes you've made you've updated both 
InputManager.cs and
InputSettingManager.cs

I assume I will then need to add the symbol USE_REWIRED to the Scripting Define Symbols to use this? I ask this because I assume rewired is the rewired plugin which I don't see any rewired files included with the "Shooter IO PUN" package etc.

Is there anything else I will need to update/add? 

Is there also a list I can return to see the currently saved list of "REWIRED" controls? Say for example the dash button which was set in the original input manager as positive button "left shift", how do I access/get (to view this to a label) and set/change this in the new rewired system?

**Comment by insthync** [comment]
If you use Rewired plugins you can set key configs in Rewired input manager and use Rewired's control remapping, don't have to use my `InputSettingManager`. Just have to add the `USE_REWIRED` symbol and use all Rewired's systems.

**Comment by Whiskah** [comment]
Edit (again)
Hmmm righty well I don't have anything like rewired packages etc and would be happy to just use your input manager, I've attached the input setting manager to a gameobject and added a keybind then run the program I've altered my code to use 
InputManager.GetButtonDown("use") which then has E as the keycode, but how do I handle the character movement which uses the character controller? Does this mean I have to add/alter something on BRCharacterEntity prefab to make this work or is there a specific name I need to use to make this work for (up(w)/down(d)/left(a)/right(d)) etc?

EDIT:
Ignore all my other edits I cannot get this working with the axis/character controller :(
